superset/assets/javascripts/explore/stores/visTypes.js

```diff
@@ -940,6 +945,8 @@ export const visTypes = {
     },
   },
 };
+visTypes.line_ttest.controlPanelSections = visTypes.line.controlPanelSections;
```

Comment:
Seems like this would be easy to miss. I'd prefer defining `const lineVis = {...}` at the top and then 
```
line: lineVis,
line_ttest: Object.assign({}, lineVis, {'label': '...'}),
```